core/modules/wsched/GroupScheduler.cc

```diff
@@ -53,7 +53,7 @@ namespace qserv {
 namespace wsched {
 
 
-GroupQueue::GroupQueue(int maxAccepted, wbase::Task::Ptr const& task) : _maxAccepted{maxAccepted} {
+GroupQueue::GroupQueue(int maxAccepted, wbase::Task::Ptr const& task) : _maxAccepted(maxAccepted) {
```

Comment:
I do. I thought that someone wanted us using parenthesis because of the curly bracket reference compiler issue, but I appear to have been mistaken.